.github/workflows/review_pull_request.yml

```
@@ -48,31 +49,31 @@ jobs:
         with:
-          ruby-version: '2.7'
-
-      - name: Cache gems
-        uses: actions/cache@v2
+          distribution: 'temurin'
+          java-version: '17'
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.0'
+          bundler-cache: true
+          run: bundle install
```

actionlint (1.7.4)

input "run" is not defined in action "ruby/setup-ruby@v1". available inputs are "bundler", "bundler-cache", "cache-version", "ruby-version", "rubygems", "self-hosted", "windows-toolchain", "working-directory"

(action)